test/ctia/http/routes/graphql/investigation_test.clj

```diff
@@ -0,0 +1,84 @@
+(ns ctia.http.routes.graphql.investigation-test
+  (:require [clj-momo.test-helpers.core :as mth]
+            [clojure.test :refer [deftest is join-fixtures testing use-fixtures]]
+            [ctia.entity.investigation :as inv]
+            [ctia.entity.investigation.examples :refer [investigation-maximal]]
+            [ctia.test-helpers.auth :refer [all-capabilities]]
+            [ctia.test-helpers.core :as helpers]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.fixtures :as fixt]
+            [ctia.test-helpers.graphql :as gh]
+            [ctia.test-helpers.store :refer [test-for-each-store]]))
+
+(use-fixtures :once (join-fixtures [mth/fixture-schema-validation
+                                    helpers/fixture-properties:clean
+                                    whoami-helpers/fixture-server]))
+
+(use-fixtures :each whoami-helpers/fixture-reset-state)
+
+(def ownership-data-fixture
+  {:owner "foouser"
+   :groups ["foogroup"]})
+
+(def investigation-1 (fixt/randomize investigation-maximal))
+(def investigation-2 (assoc
+                      (fixt/randomize investigation-maximal)
+                      :source "ngfw"))
+
+(defn select-graphql-fields
+  [investigation]
+  (select-keys investigation
+               [:id
+                :type
+                :schema_version
+                :title
+                :timestamp
+                :tlp
+                :source
+                :owner
+                :groups
+                :object_ids
+                :investigated_observables
+                :targets]))
```

Comment:
If you select the keys, the graphql test will not fail if new fields are added to the schema or the example. I would dissoc fields that will never be returned by the graphql query.